src/app/mandate/mandate.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component } from '@angular/core';
+import {ComponentsModule} from "@biggive/components-angular";
+import {DatePipe} from "@angular/common";
+import {ExactCurrencyPipe} from "../exact-currency.pipe";
+import {FaIconComponent} from "@fortawesome/angular-fontawesome";
+import {MatProgressSpinner} from "@angular/material/progress-spinner";
+import {Mandate} from "../mandate.model";
+
+@Component({
+  selector: 'app-mandate',
+  standalone: true,
+  imports: [
+    ComponentsModule,
+    DatePipe,
+    ExactCurrencyPipe,
+    FaIconComponent,
+    MatProgressSpinner
+  ],
+  templateUrl: './mandate.component.html',
+  styleUrl: './mandate.component.scss'
+})
+export class MandateComponent {
+  personId = 'donor-id';
+  complete = true;
+  encodedShareUrl = '';
+  encodedPrefilledText: string = '';
+  donation: any = {};
+  totalValue: number = 20;
+  giftAidAmount: number = 0;
+  totalPaid: 0;
+  mandate: Mandate = {
```

Comment:
I think it's fine to keep this as a hard-coded dummy for merging this, but lets just add a todo comment.

```suggestion
  /** @todo-regular-giving : replace this with a mandate fetched from matchbot *//
  mandate: Mandate = {
```